lib/screens/admin_settings_page.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:badiup/widgets/settings_listing.dart';
+import 'package:flutter/material.dart';
+
+import '../colors.dart';
```

Comment:
Please use the full path of packages. Something like: `package:badiup/colors.dart` instead of `../colors.dart`